package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Forked from Project_MEANserver, this web application is the new SCE Core, a private Github Project. Based off of the MEAN stack, the software comes packaged with a NodeJS webserver (uses ExpressJS to handle routing/request processing), the SCE organization's home/web-services page, administrator dashboard, and members portal.",
   "main": "server.js",
   "scripts": {
-    "test": "mocha",
+    "api-test": "mocha",
```

Comment:
Running both tests at the same time produced this output:
![image](https://user-images.githubusercontent.com/36345325/69607864-0a1abb80-0fdb-11ea-83c2-9f703f0667a7.png)

Splitting both api and frontend tests up mitigated this issue, and everything worked out fine.